src/basis/dom/event.js

```diff
@@ -165,6 +156,13 @@
           wheelDelta: wheelDelta(event)
         });
       }
+      else if (isTouchEvent(event))
+      {
+        extend(this, {
+          mouseX: touchX(event),
```

Comment:
`mouseX`/`mouseY` should stay for pointer/touch events for backward capability. But `pointerX`/`pointerY` should be implemented as well. And will be great if we change `mouse*` to `pointer*`